DAOS-Support.md

```diff
@@ -163,3 +155,57 @@ Number of items that exist in both directories and have different types: 0 (Src:
 Number of items that exist in both directories and have the same content: 1 (Src: 1 Dest: 1)
 Number of items that exist in both directories and have different contents: 0 (Src: 0 Dest: 0)
 ```
+
+## DAOS Serialization and Deserialization Examples
+
+daos-serialize and daos-deserialize can be used on any type of DAOS container.
+They are DAOS only tools that require HDF5.
+
+daos-serialize will serialize a DAOS container to an HDF5 file. Depending on
+the amount of data, multiple files may be written for each rank specified in the job.
+
+daos-deserialize will deserialize or restore the HDF5 files written into a new
+DAOS container, and a pool UUID to deserialize the data to must be specified.
+
+#### Example One
+
+Show the serialization of a DAOS container
+
+```shell
+$ mpirun -np 3 daos-serialize -v /$pool1/$cont1
+Serializing Container to 0-7bf8037d-823f-4fa5-ac2a-c2cae8f81f57
+Serializing Container to 1-7bf8037d-823f-4fa5-ac2a-c2cae8f81f57
+Serializing Container to 2-7bf8037d-823f-4fa5-ac2a-c2cae8f81f57
```

Comment:
I haven't used HDF a lot, but is it convention to use an .hdf extension on those?  That might help people remember the file format, especially if other formats might be supported in the future.  I'm not stuck on that, though, and we can add it later if we think it'd be helpful.

Reply:
@dsikich I think we probably should use a `.h5`  or `.hdf5` extension, per Mohamad's comment. Also, we should update the examples in this file with the `_rank` suffix you added.

Reply:
@daltonbohning this has been updated.